pkg/terraform/terraform.go

```diff
@@ -45,10 +45,10 @@ func Destroy(dir string, platform string, autoApprove bool, extraArgs ...string)
 
 	defaultArgs := []string{
 		"-input=false",
-		fmt.Sprintf("-state=%s", filepath.Join(dir, StateFileName)),
-		fmt.Sprintf("-state-out=%s", filepath.Join(dir, StateFileName)),
```

Comment:
we have given the option 

```golang
                fmt.Sprintf("-state=%s", filepath.Join(dir, StateFileName)),
		fmt.Sprintf("-state-out=%s", filepath.Join(dir, StateFileName)),
```

in line no. 25 and 26, so I'm not sure why we expecting somewhere else while the destroy!